### PR TITLE
Change industry core logos

### PR DESCRIPTION
**Pull request by Grand-Thibault**
## Description

- Industry Core is an own domain and should appear as such
- Own color scheme for IC Kits

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by jSchuetz88 — COMMENTED** [review]
Hi @Grand-Thibault,

I think you missed to update the url to the new images in ./kitsGallery.js.
Somewhere in line 20 to 38.

Additional remark:
You changed to image only for the upcoming release (R24.12). Is that on purpose? usually the gallery icons were consistent across the releases.

**Review by jSchuetz88 — APPROVED** [review]
Looks good to me.

**Comment by tom-rm-meyer-ISST** [comment]
First of all: thanks a lot! Same png / svg issue as in #1084 

Also a question to @stephanbcbauer as he may have an overview about refactorings here: Which is the targeted structure for image assets? All into the respective KIT incl. kit icons (as started here) or just putting them into the current folders.

just want to be aware of "is there a strategy"? If there is, we need to somewhat explain it in the office hours / dev mailing list.

**Comment by stephanbcbauer** [comment]
> First of all: thanks a lot! Same png / svg issue as in #1084
> 
> Also a question to @stephanbcbauer as he may have an overview about refactorings here: Which is the targeted structure for image assets? All into the respective KIT incl. kit icons (as started here) or just putting them into the current folders.
> 
> just want to be aware of "is there a strategy"? If there is, we need to somewhat explain it in the office hours / dev mailing list.

@tom-rm-meyer-ISST THX for your comment. Is it really the case SVG is not rendered any more? Since I thought this was the idea to use them. Exporting the PNG would mean we have 4 files for one image?
1. SVG as source to be able to adapt it
2. PNG Exported picture
3. License file for both?

Related to the structure, depends on if the image/icon should be versioned or not. For example, diagrams should be included in the KITs folder since the relates to specific version. If there is a static images, which are not versioned, we should keep it in the static folder. But, we should probably structure the static/img folder a little bit better?

**Comment by Grand-Thibault** [comment]
> [...] But, we should probably structure the static/img folder a little bit better?

Defininetly! As the other folders are structured mostly according to the KITs, I would do the same here.

**Comment by tom-rm-meyer-ISST** [comment]
@stephanbcbauer  exactly, same here.
@Grand-Thibault overall it's a great idea, but we need to either directly do it for all OR make everyone aware of that change, so they'll do during next PRs. I only see the issue with the latter, that it's not going to happen, if we don't announce it.

If you both prefer, the document structure n this PR, then go for it and ignore my comments in the other PR! :)

Sidenote: Please remember to remove the old images, too :)

File: static/img/kits/data-chain/DataChainKitIcon.png
Comment (tom-rm-meyer-ISST) [inline static/img/kits/data-chain/DataChainKitIcon.png:1]: png should be a svg. Please also consider license accordingly

File: static/img/kits/data-chain/data-chain-kit-gallery.drawio.license
Comment (tom-rm-meyer-ISST) [inline static/img/kits/data-chain/data-chain-kit-gallery.drawio.license:1]: filename should be `....drawio.svg.license`

File: static/img/kits/data-chain/data-chain-kit-logo.drawio.license
Comment (tom-rm-meyer-ISST) [inline static/img/kits/data-chain/data-chain-kit-logo.drawio.license:1]: filename should be `....drawio.svg.license`

File: static/img/kits/industry-core/industry-core-kit-gallery.drawio.license
Comment (tom-rm-meyer-ISST) [inline static/img/kits/industry-core/industry-core-kit-gallery.drawio.license:1]: filename should be `....drawio.svg.license`

File: static/img/kits/industry-core/industry-core-kit-logo.drawio.license
Comment (tom-rm-meyer-ISST) [inline static/img/kits/industry-core/industry-core-kit-logo.drawio.license:1]: filename should be `....drawio.svg.license`

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Please check license names

File: static/img/kits/data-chain/DataChainKitIcon.png
Comment (Grand-Thibault) [reply static/img/kits/data-chain/DataChainKitIcon.png:1]: png kept for previous version

**Pull request merged**